ladybug_comfort/et.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""Effective temperature transitioned from legacy code of @stgeorges"""
+
+def effective_temperature(Ta, ws, rh, SR, ac):
+    # inputs: (Ta, ws, rh, SR, ac):
+    if ws <= 0.2:
+        # formula by Missenard
+        TE = Ta - 0.4*(Ta - 10)*(1-rh/100)
+    elif ws > 0.2:
+        # modified formula by Gregorczuk (WMO, 1972; Hentschel, 1987)
+        TE = 37 - ( (37-Ta)/(0.68-(0.0014*rh)+(1/(1.76+1.4*(ws**0.75)))) ) - (0.29 * Ta * (1-0.01*rh))
```

Comment:
The spacing here needs to follow PEP8. As do all of the variable names:

https://peps.python.org/pep-0008/